packages/core/src/css_composer/model/CssRule.ts

```diff
@@ -117,6 +121,7 @@ export default class CssRule extends StyleableModel<CssRuleProperties> {
     this.em = opt.em;
     this.ensureSelectors(null, null, {});
     this.on('change', this.__onChange);
+    super.setStyle(this.get('style'));
```

Comment:
Note** to call `.setStyle` on the styleable model to register the data var listeners. "The magic line". We could approach this better, open to thoughts....

Reply:
I think for now it's fine